docs/sources/get-started/install/linux.md

```diff
@@ -78,6 +78,17 @@ sslcacert=/etc/pki/tls/certs/ca-bundle.crt' | sudo tee /etc/yum.repos.d/grafana.
    ```
    {{< /code >}}
 
+1. Optional: Edit the run configuration to enable the {{< param "PRODUCT_NAME" >}} UI
+
+   ```bash
+   sed -i -e 's/CUSTOM_ARGS=""/CUSTOM_ARGS="--server.http.listen-addr=0.0.0.0:12345"/' /etc/default/alloy
+   ```
+
+The {{< param "PRODUCT_NAME" >}} UI is a web application that runs on `http://localhost:12345/` and allows for
+[../../tasks/debug/](debugging your configuration) and viewing it in a visual format. Enabling this UI is recommended
```

Comment:
We should try to group all relative URI references at the bottom of the topic - for consistency more than anything. There's nothing wrong with it here, but it simplifies maintenance if they are grouped (eg reduces multiple, sometimes competing, link definitions).